Unable to build the  source tree

I  am following  the instructions to setup  the development environment using  Visual Code.
All  initial steps  to  setup development container  and run bootstrap work.

When I  get to the last step,  I  do Run Build Task ---> Main. Terminal shows it is compiling but then  stops with  an  error:

checking D-Bus configuration directory... /etc
checking D-Bus system bus services dir... /usr/share/dbus-1/system-services
checking D-Bus session bus services dir... /usr/share/dbus-1/services
${prefix}/share/zsh/site-functions
checking for UDEV... no
configure: error: libudev >= 172 is required
configure: error: ../../../../../third_party/bluez/repo/configure failed for third_party/bluez/repo
The terminal process "/bin/bash '-c', 'scripts/build/bootstrap.sh'" terminated with exit code: 1.

I  am using a Mac for development

--- a/.devcontainer/devcontainer.json
+++ b/.devcontainer/devcontainer.json
@@ -8,7 +8,7 @@
         "dockerfile": "Dockerfile",
         "args": {
             //  "BUILD_VERSION": "$(cat integrations/docker/images/chip-build/version)" // trying to get this to work
-            "BUILD_VERSION": "0.3.0"
+            "BUILD_VERSION": "0.4.0"
         }
     },
     "remoteUser": "vscode",

